pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -91,7 +92,7 @@ func (backfill *Action) Execute(ssn *framework.Session) {
 			}
 		}
 
-		klog.V(3).Infof("Binding Task <%v/%v> to node <%v>", task.Namespace, task.Name, node.Name)
+		klog.V(4).Infof("Binding Task <%v/%v> to node <%v>", task.Namespace, task.Name, node.Name)
```

Comment:
We'd better keep binding info logs, because in production env, we offer check the binding log to see the scheduling latancy.

Reply:
done